.githooks/pre-commit

```diff
@@ -57,4 +57,6 @@ done
 # If there are whitespace errors, print the offending file names and fail.
 git diff-index --check --cached $against --
 
+npm shrinkwrap --dev
```

Comment:
More generally, I'd like to fail the commit if you add a new dependency and don't shrinkwrap it. Not actually fussed as to how we do that.